src/clj/qbits/spandex.clj

```diff
@@ -202,30 +203,66 @@
                (response-headers response)
                (.getHost response))))
 
+(defn response-ex->response
```

Comment:
I think this kind of utility function should be kept even if you decide to not keep the whole exception handling.